.github/workflows/release.yml

```diff
@@ -729,12 +729,8 @@ jobs:
             fail-fast: false
             matrix:
                 chains:
-                    - chain: stagenet_dancebox
-                      runtime: dancebox
                     - chain: flashbox
                       runtime: flashbox
-                    - chain: dancebox
-                      runtime: dancebox
                     - chain: frontier_template
```

Comment:
I guess this also should be removed, because it uses dancebox URL [here](https://github.com/moondance-labs/tanssi/blob/0f92ae5e0ffdf1fd559006b2188552fcf4b7bec3/test/configs/frontierContainer.yml#L1-L0).